qa/tasks/vstart_runner.py

```diff
@@ -396,10 +396,22 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
 
         return proc
 
-    def sh(self, command, log_limit=1024, cwd=None, env=None):
+    # XXX: for compatibility keep this method same teuthology.orchestra.remote.sh
+    def sh(self, script, **kwargs):
+        """
+        Shortcut for run method.
+
+        Usage:
+            my_name = remote.sh('whoami')
+            remote_date = remote.sh('date')
+        """
+        if 'stdout' not in kwargs:
+            kwargs['stdout'] = BytesIO()
```

Comment:
stdout is StringIO instance instead of BytesIO at master - https://github.com/ceph/ceph/blame/master/qa/tasks/vstart_runner.py#L434. So, I think we should backport this commit too along - https://github.com/ceph/ceph/commit/624944503e7c6aa246331f4d9f291cf714296380.

I am not aware of how it works when we have 2 PRs with related changes that needs to be backported -- do we backport this other commit in this PR itself or do I need to request/create a separate backport?

Reply:
@smithfarm While checking [this issue](https://github.com/ceph/teuthology/pull/1459), I think it's better cherry-pick 624944503e7c6aa246331f4d9f291cf714296380 too in this PR.